webrtc.html

```diff
@@ -2481,7 +2519,17 @@
             associated, via <code>setLocalDescription</code>,
             with this <code><a>RTCRtpSender</a></code> object.
           </dd>
-        </dl>
+
+	  <dt>attribute PriorityValue priority</dt>
```

Comment:
I don't think I agree.
a) RtpEncodingParams don't have a proposal yet. That means it's at risk.
b) -transport's prioritization algorithm talks about prioritization in terms of various tracks being sent at various priorities. Burying them one level down seems like the wrong idea.

Suggest we leave it on RTP sender, but say (when RtpEncodingParams gets proposed) that it can be overridden (to a lower value only, I assume) for individual SSRCs.